src/screens/DrawerNavigator.tsx

```diff
@@ -1,40 +1,50 @@
+/* eslint-disable react-native/no-inline-styles */
 import React, {Component} from 'react';
 import {createDrawerNavigator} from '@react-navigation/drawer';
-import ChooseCurrentLocation from "./ChooseCurrentLocation";
-import {Image} from 'react-native'
-import {colors} from "../constants";
-import InternalTransfer from "./InternalTransfer";
+import ChooseCurrentLocation from './ChooseCurrentLocation';
+import {Image} from 'react-native';
+import {colors} from '../constants';
 
 const Drawer = createDrawerNavigator();
 
 class DrawerNavigator extends Component {
-    render() {
-        return (
-            <Drawer.Navigator screenOptions={{
-                headerStyle: {
-                    backgroundColor: colors.headerColor
-                },
-                headerTitleStyle:{
-                    color:"white"
-                },
-                headerTintColor: '#ffffff',
-                headerRight: () => <Image
-                    source={require('../assets/images/logo.png')}
-                    style={{resizeMode: 'stretch', width: 40, height: 30, marginRight: 30}}
-                />
-            }}>
-                <Drawer.Screen
-                    name="Choose Current Location"
-                    component={ChooseCurrentLocation}
-                    options={{
-                        // headerStyle: {
-                        //     backgroundColor: '#f4511e',
-                        // }
-                    }}
-                />
-            </Drawer.Navigator>
-        );
-    }
+  render() {
+    return (
+      <Drawer.Navigator
+        screenOptions={{
+          headerStyle: {
+            backgroundColor: colors.headerColor,
+          },
+          headerTitleStyle: {
+            color: 'white',
+          },
+          headerTintColor: '#ffffff',
+          headerRight: () => (
+            <Image
+              source={require('../assets/images/logo.png')}
+              style={{
```

Comment:
Move to styles

Reply:
@awalkowiak those all changes already exit in a restructures branches. i have not made any changes.

Reply:
@hellochirag yeah I know, but if there is a need to add an `eslint-disable` rule comment for something that is easy to fix it is better to fix it.